apps/script/checkEntryPoints.js

```diff
@@ -111,7 +111,8 @@ const SITES_CONFIG = {
   }
 };
 
-const ENTRY_POINT_FILE_PATH_PATTERN = /^\.\/src\/sites\/([\w.]+)\/pages\/(.*)\.jsx?$/;
+const ENTRY_POINT_FILE_PATH_PATTERN =
```

Comment:
not sure why the linter started complaining about this in this pr specifically, but these changes align with our expected linter behavior.